lib/spack/spack/cmd/style.py

```diff
@@ -242,12 +252,9 @@ def run_flake8(flake8_cmd, file_list, args):
     # run in chunks of 100 at a time to avoid line length limit
     # filename parameter in config *does not work* for this reliably
     for chunk in grouper(file_list, 100):
-
         output = flake8_cmd(
-            # use .flake8 implicitly to work around bug in flake8 upstream
-            # append-config is ignored if `--config` is explicitly listed
-            # see: https://gitlab.com/pycqa/flake8/-/issues/455
-            # "--config=.flake8",
```

Comment:
@trws: is this still an issue?  I could not see where it was.

Reply:
It mattered when we had different flake8 settings for the repo and the rest of spack, but I think that's gone now so the append-config option is also gone.